src/test/kotlin/net/emdal/tank/CreateQueryTest.kt

```diff
@@ -0,0 +1,80 @@
+package net.emdal.tank
+
+import org.junit.jupiter.api.Disabled
+import org.junit.jupiter.api.Test
+
+class CreateQueryTest {
+
+  @Test
+  fun `create empty node`() {
+    val query = Graph().create {
+      node()
+    }.query
+
+    query eq """
+      CREATE ()
+    """.trimIndent()
+  }
+
+  @Test
+  internal fun `create node with alias`() {
+    val query = Graph().create {
+      node("n")
+    }.query
+
+    query eq """
+      CREATE (n)
+    """.trimIndent()
+  }
+
+
+  // TODO: Discuss multiple node creation syntax
+  @Disabled
+  @Test
+  internal fun `create multiple nodes with different aliases`() {
+    val query = Graph().create {
+      node("n")
+      node("m")
+    }.query
+
+    query eq """
+      CREATE (n), (m)
+    """.trimIndent()
```

Comment:
I think we have several options here:
```
CREATE (n)
CREATE (m)
```
might have the same result as:
```
CREATE (n), (m)
```

I think this test will pass already if it were rewritten to:
```
    val query = Graph().create {
      node("n")
      .node("m")
    }.query

    query eq """
      CREATE (n)
      CREATE (m)
    """.trimIndent()
```